app/models/spot.rb

```diff
@@ -0,0 +1,12 @@
+class Spot < ActiveRecord::Base
+  has_many :reviews
+  belongs_to :user
+  validates :name, presence: true
+  validates :description, presence: true
+  validates :category, presence: true
+  validates :address, presence: true, uniqueness: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zip_code, presence: true, length: { is: 5 }
+  validates :phone, presence: true, length: { minimum: 10 }
```

Comment:
Do we want to validate the presence of all of these fields? I'm thinking that validating the phone number isn't really necessary.